rep-2001.rst

```diff
@@ -773,6 +775,33 @@ Desktop Full
       packages: [perception, simulation, ros_ign_gazebo_demos]
 
 
+ROS Distribution Independent (September 2022 - Ongoing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These variants are not released as ROS packages within each distribution as the others are.
+Instead they are made available as independent packages outside of any ROS distribution.
+
+Build Essential
+"""""""""""""""
+
+::
+
+ - ros-build-essential:
+ - ros-build-essential:
+      packages: [cmake, git, python3-setuptools, wget]
+
+Dev Tools
+"""""""""
+
+::
+
+  - ros-dev-tools:
+      extends:  [ros-build-essential]
+      packages: [python3-bloom, python3-colcon-common-extensions,
+                 python3-rosdep, python3-vcstool]
```

Comment:
Move `wget` to the dev-tools variant instead.

```suggestion
      packages: [python3-bloom, python3-colcon-common-extensions,
                 python3-rosdep, python3-vcstool, wget]
```

Reply:
[91f54b3](https://github.com/ros-infrastructure/rep/pull/355/commits/91f54b31cd23e368829914d4cbc9d6a696f4f0b5)